test-bdd/main_test.go

```diff
@@ -76,6 +76,8 @@ func TestE2e(t *testing.T) {
 	RunSpecsWithDefaultAndCustomReporters(t, "InstanceGroup Type Suite", []Reporter{junitReporter})
 }
 
+
+
```

Comment:
seems like we can undo these extra blank lines

Reply:
Quick question. What files besides the .idea files would you want removed?